packages/loot-design/src/components/modals/PlaidExternalMsg.js

```diff
@@ -130,10 +130,7 @@ export default function PlaidExternalMsg({
             Typing your bank{"'"}s username and password is one of the most
             security-sensitive things you can do, and the browser is the most
             secure app in the world. Why not use it to make sure your
-            information is safe?{' '}
-            <a href="https://actualbudget.com/security-learn-more">
-              Learn more about security
-            </a>
+            information is safe?
```

Comment:
This whole message needs to be rewritten at some point given we’re now browser-first but in the meantime it should be clear to anyone working on this that we need an actual (ha) explanation:

```suggestion
            information is safe? [TODO: Link to docs article]
```